class/Services/TemplateService.php

```diff
@@ -24,6 +24,8 @@ public static function add_preview_to_hierarchy($original_template)
       return get_query_template("single-passle-post");
     } else if (get_query_var("post_type") == "passle-post") {
       $is_passle_preview = false;
+	  $shortcode = get_query_var("name");
+	  $post = PasslePostsContentService::fetch_preview($shortcode);
```

Comment:
We shouldn't be fetching the post from the API here - if it's not a preview we should be using the synced content, otherwise we're making an API call every time someone views a post.

`$post` is a global which in this case should contain the Wordpress post instance of the synced Passle post.

Reply:
Realised after recent testing that this change is not needed, so reverted it in future commit